kotlin/android/app/src/main/java/dev/firezone/android/features/session/backend/SessionCallbackImpl.kt

```diff
@@ -1,16 +1,24 @@
 package dev.firezone.android.features.session.backend
 
 import android.util.Log
+import dev.firezone.connlib.session.SessionCallback
 
-public object SessionCallbacks {
-    public fun onUpdateResources(resources: String): Boolean {
+class SessionCallbackImpl: SessionCallback {
+
+    override fun onConnect(status: String): Boolean {
```

Comment:
`onSetTunnelAddresses` was renamed to `onConnect`. See https://github.com/firezone/product/issues/586 for more info.

```suggestion
    override fun onConnect(addresses: String): Boolean {
```